packages/merge-styles/src/styleToClassName.ts

```diff
@@ -23,8 +23,8 @@ function getDisplayName(rules?: { [key: string]: IRawStyle }): string | undefine
 }
 
 function expandSelector(newSelector: string, currentSelector: string): string {
-  if (newSelector.indexOf(':global(') === 0) {
-    return newSelector.replace(/:global\(|\)$/g, '');
+  if (newSelector.indexOf(':global(') >= 0) {
+    return newSelector.replace(/:global\((.+?)\)/g, '$1');
```

Comment:
I'm ok with this, but just out of curiosity, wouldn't `& button` do the same thing?

Reply:
Ah yes it does actually, while i should have known that. I never seemed to have realised that `:global()` is totally optional (even though I've seen it being optional in before without having realized it) :D

@jakob101 is trying to write a one time converter script from less to mergeStyles, and we have a bunch of these `:global` selectors so we ran into this. But since it's optional we might as well drop `:global` if that makes it easier.